visual_testing_wdio/package.json

```diff
@@ -0,0 +1,25 @@
+{
+  "name": "visualcomparisonblog",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "wdio": "wdio run ./wdio.conf.js",
+    "test-mobile": "wdio run wdio.chrome.mobile.conf.js --params.deviceType=${npm_config_device_type} --params.deviceName=${npm_config_device_name} --suite ${npm_config_suite}"
+  },
+  "author": "",
+  "license": "ISC",
+  "dependencies": {
+    "-": "^0.0.1",
+    "@wdio/cli": "^8.18.2",
+    "save-dev": "^0.0.1-security"
+  },
+  "devDependencies": {
+    "@wdio/jasmine-framework": "^8.18.2",
```

Comment:
Use fixed versions for dependencies to avoid some breaking change that might happen down the line in dependencies update or update of subdependencies